src/terminal/adapter/adaptDispatch.cpp

```diff
@@ -2138,11 +2103,7 @@ bool AdaptDispatch::RequestMode(const DispatchTypes::ModeParams param)
         state = mapTemp(_pages.ActivePage().Cursor().IsVisible());
         break;
     case DispatchTypes::ModeParams::XTERM_EnableDECCOLMSupport:
```

Comment:
have we tested DECCOLM over new-conpty?

Reply:
If you "Load more..." comments you can find a brief discussion between James and me about DECCOLM. To summarize, James believes that it's not a big deal and recommended that we remove it from the DA1 report, which I did.